packages/suite/src/actions/wallet/addWalletThunk.ts

```diff
@@ -16,10 +16,23 @@ import { getBackgroundRoute } from 'src/utils/suite/router';
 import { getDiscoveryStatus } from '../../utils/wallet/getDiscoveryStatus';
 import { goto } from '../suite/routerActions';
 
-export const redirectAfterWalletSelectedThunk = createThunk<void, void, void>(
+export const redirectAfterWalletSelectedThunk = createThunk<
+    void,
+    { forceDeviceDashboard?: boolean },
+    void
+>(
     `${DEVICE_MODULE_PREFIX}/redirectAfterWalletSelectedThunk`,
-    async (_, { dispatch }) => {
+    async ({ forceDeviceDashboard }, { dispatch }) => {
```

Comment:
This thunk was written by @seibei-iguchi I am not sure what it was suppose to do though.

Reply:
Yeah, in some cases, it went to dashboard.